src/anbox/bridge/platform_api_skeleton.cpp

```diff
@@ -22,6 +22,8 @@
 #include "anbox/wm/window_state.h"
 #include "anbox/logger.h"
 
+#undef Status
```

Comment:
https://github.com/anbox/anbox/pull/29/commits/9c221215a1d7f80ada1da1e59797a3b249fea03c looks good to me